src/AltGoslingComponent.tsx

```diff
@@ -44,6 +44,7 @@ interface AltGoslingCompProps extends GoslingCompProps {
     layoutPanels?: 'vertical' | 'horizontal';
     download?: boolean;
     dataTableRoundValues?: boolean;
+    simplifyColor?: boolean;
```

Comment:
I would slightly prefer to be more explicit about what this is changing (i.e., color names):

```suggestion
    simplifyColorNames?: boolean;
```